### DISCUSSION TITLE
CC2652RB coordinator with CC2530/CC2531 routers

### DISCUSSION THREAD

**Discussion by florentin68** [Koenkk/zigbee2mqtt#6814] [Devices]
Hello,

I've just received my new CC2652BR (slaesh) ZigBee stick which runs Z-Stack 3.x.0.
I would like to know if my CC2531 coordinator can be switched to a router (with Z-Stack 1.2) and if my existing CC2530 router will still work after reflashing (to pair to the new coordinator)?

Thank you in advance for your help.

**Comment by frank3523** [comment]
I also have the slae stick and also use a CC2530 as a router.
Works fine no problems.
